src/serviceorchestration/5_0_0/sd/service-orchestration-push-management/service_orchestration_push_management_sd.tex

```diff
@@ -0,0 +1,523 @@
+origin & \pref{String} & Origin of the error. \\ \hline
+\end{tabularx}
+\end{table}
+
+\msubsection{struct}{OrchestrationUnsubscribeListRequest}
+
+\begin{table}[ht!]
+\begin{tabularx}{\textwidth}{| p{2.5cm} | p{6.3cm} | p{2cm} | X |} \hline
+\rowcolor{gray!33} Field & Type & Mandatory & Description \\ \hline
+authentication & \hyperref[sec:model:Identity]{Identity} & yes & The requester of the operation. \\ \hline
+subscriptionIds &  \pref{List}$<$\hyperref[sec:model:OrchestrationSubscriptionId]{OrchestrationSubscriptionId}$>$ & yes & List of subscription identifiers. \\ \hline
+\end{tabularx}
+\end{table}
+
+\clearpage
+
+\msubsection{struct}{OrchestrationPushTriggerRequest}
+
+\begin{table}[ht!]
+\begin{tabularx}{\textwidth}{| p{2.5cm} | p{6.3cm} | p{2cm} | X |} \hline
+\rowcolor{gray!33} Field & Type & Mandatory & Description \\ \hline
+authentication & \hyperref[sec:model:Identity]{Identity} & yes & The requester of the operation. \\ \hline
+targetSystems &  \pref{List}$<$\pref{SystemName}$>$ & no & List of target consumer systems. \\ \hline
+subscriptionIds &  \pref{List}$<$\hyperref[sec:model:OrchestrationSubscriptionId]{OrchestrationSubscriptionId}$>$ & no & List of subscription identifiers. \\ \hline
+\end{tabularx}
+\end{table}
+
+\msubsection{struct}{OrchestrationPushJobListResponse}
+
+\begin{table}[ht!]
+\begin{tabularx}{\textwidth}{| p{3cm} | p{6.5cm} | X |} \hline
+\rowcolor{gray!33} Field & Type & Description \\ \hline
+jobs & \pref{List}$<$\hyperref[sec:model:OrchestrationJobResponse]{OrchestrationJobResponse}$>$ & List of orchestration push jobs. \\ \hline
+\end{tabularx}
+\end{table}
+
+\msubsection{struct}{OrchestrationJobResponse}
+
+\begin{table}[ht!]
+\begin{tabularx}{\textwidth}{| p{2.8cm} | p{4.5cm} | X |} \hline
+\rowcolor{gray!33} Field & Type & Description \\ \hline
+id & \hyperref[sec:model:OrchestrationJobId]{OrchestrationJobId} & Unique job identifier. \\ \hline
+status & \pref{OrchestratioJobStatus} & Actual working state of the job. \\ \hline
+type & \pref{OrchestrationType} & Type of orchestration. \\ \hline
+requesterSystem & \pref{SystemName} & Name of the system that started the orchestration process. \\ \hline
+targetSystem & \pref{SystemName} & Name of the system for which the orchestration is executed. \\ \hline
+serviceDefinition & \pref{ServiceName} & Name of the service that the orchestration job is targeting. \\ \hline
+subscriptionId & \hyperref[sec:model:OrchestrationSubscriptionId]{OrchestrationSubscriptionId} & Unique identifier of associated subscription record. \\ \hline
+message & \pref{String} & Additional error or warning information. \\ \hline
+createdAt & \pref{DateTime} & The job was created at this timestamp. \\ \hline
+startedAt & \pref{DateTime} & The job was started at this timestamp. \\ \hline
+finishedAt & \pref{DateTime} & The job was finished at this timestamp. \\ \hline
+\end{tabularx}
+\end{table}
+
+\msubsection{struct}{OrchestrationJobId}
```

Comment:
should be primitive. If you moved this one to the primitives section, you must change the references as well from hyperref to pref.